starfish/imagestack/imagestack.py

```diff
@@ -434,8 +434,8 @@ def set_slice(
         data : np.ndarray
             The source data for the operation
         axes : Optional[Sequence[Indices]]
-            The order of the axes for the source data, excluding (Y, X).  If not provided, it is
-            assumed that the data is a 2D tile.
+            The order of the axes for the source data, excluding (Y, X). Optional ONLY if data
```

Comment:
the wording of this seems to presume that X, Y tiles are supported.  I don't believe that is the case.  can this be further clarified that it requires Y, X tiles?

Reply:
Updated & re-requested.